packages/labs/cli/src/test/gen/react_test.ts

```diff
@@ -47,13 +48,14 @@ test('basic wrapper generation', async ({rig, testConsole}) => {
       rig.rootDir,
     ],
     {
+      cwd: rig.rootDir,
       console: testConsole,
     }
   );
   testConsole.alsoLogToGlobalConsole = true;
   await cli.run();
 
-  assert.equal(testConsole.errorStream.buffer.length, 0);
+  assert.equal(testConsole.errorStream.buffer.join(''), '');
```

Comment:
Minor/for my understanding: Unclear how these would be different? I guess the writeable could get a 0-char chunk? Should we just make a `toString()` on `BufferedWriteable` that does the join?

I guess I'm unclear on whether there is semantic meaning to chunk boundaries such that anyone would want the array, or if chunk boundaries could be arbitrary and thus the `buffer` array should be private and callers should only ever get the `toSring()` result?